tofu/util.py

```diff
@@ -86,7 +86,7 @@ def positive_int(value):
     return result
 
 
-def tupleize(num_items=None, conv=None):
+def tupleize(num_items=None, conv=float):
```

Comment:
Now `tupleize()('1,2,3')` returns `[1.0, 2.0, 3.0]`.

Reply:
...which is how I will often use it for the reconstruction params.

Reply:
Okay, okay :+1: